balances-adjustment/src/main/java/com/powsybl/balances_adjustment/util/Reports.java

```diff
@@ -0,0 +1,83 @@
+/**
+ * Copyright (c) 2023, Coreso SA (https://www.coreso.eu/) and TSCNET Services GmbH (https://www.tscnet.eu/)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.balances_adjustment.util;
+
+import com.powsybl.commons.reporter.Report;
+import com.powsybl.commons.reporter.Reporter;
+import com.powsybl.commons.reporter.TypedValue;
+
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * @author George Budau {@literal <george.budau at artelys.com>}
+ */
+public final class Reports {
+
+    private static final String ITERATION = "iteration";
+    private static final String AREA_NAME = "areaName";
+
+    private Reports() {
+    }
+
+    public static void reportScaling(Reporter reporter, String areaName, double offset, double done) {
+        reporter.report(Report.builder()
+                .withKey("areaScaling")
+                .withDefaultMessage("Scaling for area ${areaName}: offset=${offset}, done=${done}")
+                .withValue(AREA_NAME, areaName)
+                .withValue("offset", offset)
+                .withValue("done", done)
+                .withSeverity(TypedValue.INFO_SEVERITY)
+                .build());
+    }
+
+    public static void reportConvergenceError(Reporter reporter) {
+        reporter.report(Report.builder()
+                .withKey("convergenceError")
+                .withDefaultMessage("LoadFlow on network does not converge")
```

Comment:
"Load flow" for coherence. Note that in general a load flow does not converged on a synchronous component number with open-loadflow. You must be more accurate here.

Reply:
we should probably put the iterationReporter in the context, and move the loadflow subreport creation inside the `isLoadFlowResultOk` method

Reply:
@annetill @jeandemanged 

I've added the status of the convergence of the load flow, could you please check if it's in the right format?